dev/public/index.html

```diff
@@ -23,11 +23,12 @@
     </nav>
     <!-- Page section -->
     <div class="flex flex-col md:flex-row flex-grow">
-      <!-- Left Navigation Panel -->
+      <!-- Control Panel -->
       <div class="bg-gray-800 shadow-xl w-full md:w-48">
         <ul class="flex flex-row md:flex-col py-0 px-1 md:px-2 md:mr-3">
             <li>
-              <div class="select-none py-1 md:pb-5 md:pt-3 pl-1 border-b-2 border-gray-800">
+              <!-- TODO if needed, to align the button with the container, replace md:pt-3 by md:pt-0 -->
```

Comment:
ℹ️ apply or remove depending if the 'bpmn upload button' should be aligned with the diagram container

Reply:
Ok to align the 'bpmn upload button' with the diagram container

Reply:
done in 65039d0